org.sdrangel.SDRangel.yaml

```diff
@@ -0,0 +1,349 @@
+        sha256: 5a2d5ca37cef3b6deddd5ce8c73918f27936c50eb0e63b27e4b4fc493310518d
+
+  - name: serialdv
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+    sources:
+      - type: archive
+        url: https://github.com/f4exb/serialDV/archive/refs/tags/v1.1.4.tar.gz
+        sha256: 0764701ac7b52ab4e7db26d91b0c0d5206377574a8ce4d705c9ec99b8f85ef79
+
+  - name: dsdcc
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+      - -DUSE_MBELIB=ON
+    sources:
+      - type: archive
+        url: https://github.com/f4exb/dsdcc/archive/refs/tags/v1.9.3.tar.gz
+        sha256: c87f7db419c12ca43f2af58feec4efcb4a8164b32e51474ac1b797338ce5695a
+
+  - name: codec2
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -Wno-dev
+    sources:
+      - type: archive
+        url: https://github.com/drowe67/codec2/archive/refs/tags/v1.0.5.tar.gz
+        sha256: cd9a065dd1c3477f6172a0156294f767688847e4d170103d1f08b3a075f82826
+
+  - name: airspy
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+    sources:
+      - type: archive
+        url: https://github.com/airspy/airspyone_host/archive/refs/tags/v1.0.10.tar.gz
+        sha256: fcca23911c9a9da71cebeffeba708c59d1d6401eec6eb2dd73cae35b8ea3c613
+
+  - name: airspyhf
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+      - -DINSTALL_UDEV_RULES=OFF
+    sources:
+      - type: archive
+        url: https://github.com/airspy/airspyhf/archive/refs/tags/1.6.8.tar.gz
+        sha256: cd1e5ae89e09b813b096ae4a328e352c9432a582e03fd7da86760ba60efa77ab
+
+  - name: rtlsdr
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+      - -DDETACH_KERNEL_DRIVER=ON
+    sources:
+      - type: archive
+        url: https://github.com/librtlsdr/librtlsdr/archive/refs/tags/v0.8.0.tar.gz
+        sha256: ce18cd70c92d44e81e804b6b4dcd655e0e370d7c8a9c59c6a8fe59e60d3447fc
+
+  - name: plutosdr
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+      - -DINSTALL_UDEV_RULE=OFF
+      - -DHAVE_DNS_SD=OFF
+    sources:
+      - type: archive
+        url: https://github.com/analogdevicesinc/libiio/archive/refs/tags/v0.24.tar.gz
+        sha256: a2b5d848531ea64fd9f95327dfd5a588bd227d9577281ec375e822702c6a52d5
+
+  - name: hackrf
+    buildsystem: cmake-ninja
+    subdir: host
+    config-opts:
+      - -Wno-dev
+      - -DINSTALL_UDEV_RULE=OFF
+    sources:
+      - type: archive
+        url: https://github.com/greatscottgadgets/hackrf/archive/refs/tags/v2021.03.1.tar.gz
+        sha256: 84a9aef6fe2666744dc1a17ba5adb1d039f8038ffab30e9018dcfae312eab5be
+
+  - name: limesdr
+    buildsystem: cmake-ninja
+    config-opts:
+      - -Wno-dev
+      - -DENABLE_QUICKTEST=OFF
+      - -DENABLE_GUI=OFF
+      - -DENABLE_SOAPY_LMS7=OFF
+      - -DENABLE_EXAMPLES=OFF
+      - -DENABLE_UTILITIES=OFF
+      - -DENABLE_HEADERS=ON
+      - -DENABLE_SIMD_FLAGS=SSE3
```

Comment:
here you force SSE. Which is it doesn't build on aarch64.

(tbf the CMakeFile should just stop here and throw an error)

Reply:
Thanks!